src/api/Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM node:18.9-alpine3.16
 
 # Variables
 ARG GITHUB_SHA
-ENV GITHUB_SHA=$GITHUB_SHA
+ARG GOV_NOTIFY_API_KEY
+ENV GITHUB_SHA $GITHUB_SHA
+ENV GOV_NOTIFY_API_KEY $GOV_NOTIFY_API_KEY
```

Comment:
we don't have these ENV as part of Dockerfile in DTFS. do we need them here? or can they be used as part of docker-compose.yml file?

Reply:
We do need them here as these are build environment variables and docker compose are only passed during runtime.

Reply:
Can we pass the values in the docker-compose.yml file? like this?
```yml
    environment:
      PORT: 5001
      DATABASE_URL: ${DATABASE_URL}
      GOV_NOTIFY_API_KEY: ${GOV_NOTIFY_API_KEY}
      COMPANIES_HOUSE_API_URL: $GOV_NOTIFY_API_KEY}
      COMPANIES_HOUSE_API_KEY: ${COMPANIES_HOUSE_API_KEY}
```

this ^ should pick up the values from .env file and pass it to the Docker image.